src/dialog/dlgdevelopertools.h

```diff
@@ -21,6 +21,7 @@ class DlgDeveloperTools : public QDialog, public Ui::DlgDeveloperTools {
     void slotControlSearch(const QString& search);
     void slotLogSearch();
     void slotControlDump();
+    void slotOscQuerryDump();
```

Comment:
typo is repeated multiple times, please `grep` for the other occurrences
```suggestion
    void slotOscQueryDump();
```